addons/sourcemod/scripting/szf/pickupweapons.sp

```diff
@@ -88,13 +90,21 @@ public Action Event_WeaponsRoundStart(Event event, const char[] name, bool dontB
 			{
 				SetRandomWeapon(iEntity, WeaponRarity_Rare);
 			}
+			case WeaponType_Common:
+			{
+				SetRandomWeapon(iEntity, eWeaponsRarity_Common);
+			}
+			case WeaponType_Uncommon:
+			{
+				SetRandomWeapon(iEntity, eWeaponsRarity_Uncommon);
+			}
```

Comment:
cool cool github not doing a build test. `eWeaponsRarity_` should be `WeaponsRarity_`.